openpype/tools/project_manager/project_manager/model.py

```diff
@@ -456,12 +582,25 @@ def add_items(self, items, parent=None, start_row=None):
         return indexes
 
     def add_item(self, item, parent=None, row=None):
+        """Add single item into model."""
         result = self.add_items([item], parent, row)
         if result:
             return result[0]
         return None
 
     def remove_delete_flag(self, item_ids, with_children=True):
+        """Remove deletion flag on items with matching ids.
+
+        Flag is also removed on all parents of passed children as it wouldn't
+        make sence to not to do so.
```

Comment:
Typo

Reply:
this could be refrased to 

```
Remove the deletion flag on items with corresponding identifiers.

The flag is also taken away from all parents of passing children because it makes no sense not to do so.
```

Reply:
> this could be refrased to

Agree with all except `corresponding identifiers` and `passing children`.